### ISSUE TITLE
Trojitá uses absurd font for various text (e.g. About page)

### ISSUE THREAD

**Issue by LianBg** [haikuports/haikuports#6426]
Heya,

I have installed the [linja pona font](https://github.com/janSame/linja-pona/) in OTF format, which is a logographic font for the Toki Pona language. Now, the problem is, for some reason, the About page in the mail client Trojitá is using this font for the credits. Some various other similar text elements have the same issue.

[Screenshot of Haiku font selector showing that I have not set the wrong font anywhere](https://user-images.githubusercontent.com/92891614/142853312-08124c72-56b4-40e0-869b-8733543b29f2.png)
This screenshot proves I have not set the linja pona font as any kind of system font.

[Screenshot of the about page of Trojitá using the wrong font](https://user-images.githubusercontent.com/92891614/142854360-0c55dd2d-554b-461e-a474-a02cb5075478.png)
This screenshot is a screenshot of the issue.

**Comment by diversys** [comment]
cc @threedeyes 
This is likely a problem with Qt5 QPA module.

**Comment by threedeyes** [comment]
Is it a problem only with this app?

**Comment by LianBg** [comment]
> Is it a problem only with this app?
@threedeyes 

Yeah, it is as far as I can see. I have tried some other apps and none of them seem to have this issue. I can't say for sure, though.

**Comment by LianBg** [comment]
> Is it a problem only with this app?

Update:
It is a problem in most to all Qt apps.

There's more information in the comment section of this ticket:
https://github.com/OtterBrowser/otter-browser/issues/1697

In short, Qt selects wrong fonts for various applications.

**Comment by humdingerb** [comment]
We had a similar issue reported as #2266 (closed).
